sapi/cli/config.w32

```diff
@@ -8,6 +8,7 @@ if (PHP_CLI == "yes") {
 	ADD_FLAG("LIBS_CLI", "ws2_32.lib");
 	ADD_FLAG("LIBS_CLI", "shell32.lib");
 	ADD_FLAG("LDFLAGS_CLI", "/stack:67108864");
+	PHP_INSTALL_HEADERS("sapi/cli", "cli.h");
```

Comment:
for consistency, would the unix version work with `PHP_INSTALL_HEADERS([sapi/cli], [cli.h])` ?

Reply:
It works also like this, yes.

Reply:
I've appended also CS commit...